.chloggen/k8sobjects.yaml

```diff
@@ -0,0 +1,12 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+# The name of the component, or a single word describing the area of concern, (e.g. agent, clusterReceiver, gateway, operator, chart, other)
+component: clusterReceiver
+# A brief description of the change. Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Add k8sobjects [cronjobs, daemonsets, deployments, jobs, namespaces, nodes, pods, replicasets, statefulsets] logs under watch mode
```

Comment:
I would also add information that collecting these objects will be enabled by default.